security/pkg/server/ca/server.go

```diff
@@ -68,13 +82,34 @@ func (s *Server) CreateCertificate(ctx context.Context, request *pb.IstioCertifi
 		s.monitoring.AuthnError.Increment()
 		return nil, status.Error(codes.Unauthenticated, "request authenticate failure")
 	}
-	// TODO: Call authorizer.
+	// By default, we will use the callers identity for the certificate
+	sans := caller.Identities
 	crMetadata := request.Metadata.GetFields()
+	impersonatedIdentity := crMetadata[security.ImpersonatedIdentity].GetStringValue()
```

Comment:
Can you give an example of the ImpersonatedIdentity?

Reply:
Its a spiffe SAN